cypress/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@tsconfig/cypress/tsconfig.json",
+
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Cypress",
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5", "dom"],
+    "types": ["cypress"]
+  },
+
+  "include": ["../next-env.d.ts", "../**/*.ts", "../**/*.tsx", "../pages/dashboard/monitoring"],
```

Comment:
Aren't we using typescript in this project? Thats why I use `tsconfig` instead of `jsconfig`. Then, `pages/dashboard/monitoring` is referencing from `tsconfig` on root directory.

Reply:
yes we're using typescript, but for the cypress directory we use javascript ([check wargabantuwarga.com](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/cypress/e2e/provinces/search.js)) because on cypress directory especially on the e2e directory doesn't have the benefit of using typescript, I'm mean it just create cypress test, and the `jsconfig` file is just to help the editor know that we will be using types from cypress, so the editor can show autocomplete

Reply:
Ah, yes. Thanks for pointing that out. I've update the file, please take a look.